### ISSUE TITLE
Way to add Title above Quit in Windows demo?

### ISSUE THREAD

**Issue by lonnietc** [fyne-io/systray#57]
Hello,

I here a way to add a "Title" that you cannot move the mouse over above the "Quit" in the example?

Thanks

**Comment by lonnietc** [comment]
Also, what about changing Text Fonts as well as Text Color foreground and background for a menu item?

**Comment by andydotxyz** [comment]
> Hello,
> 
> 
> 
> I here a way to add a "Title" that you cannot move the mouse over above the "Quit" in the example?
> 
> 
> 
> Thanks

Do you mean a tooltip?

**Comment by matwachich** [comment]
I understand something like this![Screenshot_20230909_210439_com.brave.browser.jpg](https://github.com/fyne-io/systray/assets/1791230/10c09866-789b-4d98-b9e6-954386b44682)

**Comment by lonnietc** [comment]
Yes, like the above is what I am seeking to do from the systray. I am guessing that the above image was not down with the Systray / Fyne library, but that would be awesome for my needs.

I will have a "Status" header that will show GREEN text if connected to the network, and RED if disconnected as one of the indicators, plus some others as well.

Thanks again and have a great day.

**Comment by andydotxyz** [comment]
Support for custom content is not included because some OS don't support putting that in the menu structure.

You could use this library to handle just the icon and pop up a custom window which contains anything you want though...

**Comment by lonnietc** [comment]
Thanks and I will look into that more.
